week-1/K-functions-parameters/exercise3.js

```diff
@@ -1,5 +1,8 @@
 // Write your function here
```

Comment:
Yep, the function is right!  There's a spelling error in your English, though :)